dev.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+import http.server
+import queue
+import sys
+import threading
+from pathlib import Path
+from queue import SimpleQueue
+from typing import Any
```

Comment:
tbh this seems really weird to import typing just for two usages of Any which are on an `*args` param anyway